pkg/ibmvpc/analysis_output_test.go

```diff
@@ -515,6 +515,13 @@ var tests = []*vpcGeneralTest{
 		grouping:    true,
 		format:      vpcmodel.Text,
 	},
+	// grouping test of identical names different resources and thus different UIDs that should not be grouped
+	{
+		inputConfig: "tgw_basic_example_dup_subnet_names",
+		useCases:    []vpcmodel.OutputUseCase{vpcmodel.AllSubnets},
+		grouping:    true,
+		format:      vpcmodel.Text,
+	},
```

Comment:
Is this test supposed to fail before the change in the grouping code? Isn't the grouping prevented for cross-vpc subnets anyway?

Reply:
No, grouping is not prevented in cross-vpc subnets:
E.g.:
```
Connectivity between VPCs connected by TGW local-tg-ky (UID: crn:578)
subnet11-ky,subnet12-ky => subnet21-ky : All Connections
subnet21-ky => subnet11-ky,subnet12-ky : All Connections
subnet21-ky => subnet31-ky,subnet32-ky : All Connections
subnet31-ky => subnet11-ky,subnet12-ky : All Connections *
subnet31-ky,subnet32-ky => subnet21-ky : All Connections
```

Reply:
Done, I also verified that the test fails in main.

Btw, this issue is relevant only to subnets with the same name within the same vpc. Perhaps we should have given it lower priority. Anyways now its fixed.